File: src/main/java/com/matag/admin/game/findactive/FindActiveGameController.java
package com.matag.admin.game.findactive;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import lombok.AllArgsConstructor;

@RestController
@RequestMapping("/game")
@AllArgsConstructor
public class FindActiveGameController {
  private final FindGameService findGameService;

  @GetMapping
  public ActiveGameResponse findActiveGame() {
    return findGameService.findActiveGame();
  }
}

File: src/main/java/com/matag/admin/game/findactive/FindActiveGameController.kt
package com.matag.admin.game.findactive

import org.springframework.security.access.prepost.PreAuthorize
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/game")
open class FindActiveGameController(
    private val findGameService: FindGameService
) {

    @PreAuthorize("hasAnyRole('USER', 'GUEST')")
    @GetMapping
    open fun findActiveGame(): ActiveGameResponse =
        findGameService.findActiveGame()
}
